addons/hw_drivers/iot_handlers/drivers/PrinterDriver.py

```diff
@@ -165,6 +165,12 @@ def send_status(self):
     def print_raw(self, data):
         process = subprocess.Popen(["lp", "-d", self.device_identifier], stdin=subprocess.PIPE)
         process.communicate(data)
+        
+        if process.returncode == 0:
+            return True
+        # The stderr isn't meaningful so we don't log it ('No such file or directory')
+        _logger.error('Error while printing: please check if "%s" matches the name provided on CUPS.', self.device_identifier)
```

Comment:
The intention is nice, but I don't think it would be a good idea to have this kind of error message.
Let's assume we don't use CUPS anymore (or that we use it for Windows IoT box, which does not use CUPS), your message will be misleading.
I think it would be better to just point the fact that we know. Something like:
`Printing failed: printer with the identifier "%s" could not be found`

Reply:
I totally agree with this, thanks for the input!